src/org/zaproxy/zap/extension/autoupdate/ExtensionAutoUpdate.java

```diff
@@ -817,28 +815,16 @@ private ZapXmlConfiguration getRemoteConfigurationUrl(String url) throws
     	ZapXmlConfiguration config = new ZapXmlConfiguration();
     	config.setDelimiterParsingDisabled(true);
     	config.load(new StringReader(msg.getResponseBody().toString()));
-
-        // Save version file so we can report new addons next time
+	
+		// Save version file so we can report new addons next time
```

Comment:
White space changes should be separate.
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting